catalog/to-consul/resource_test.go

```diff
@@ -256,26 +258,9 @@ func TestServiceResource_externalIP(t *testing.T) {
 	defer closer()
 
 	// Insert an LB service
-	_, err := client.CoreV1().Services(metav1.NamespaceDefault).Create(&apiv1.Service{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "foo",
-		},
-
-		Spec: apiv1.ServiceSpec{
-			Type:        apiv1.ServiceTypeLoadBalancer,
-			ExternalIPs: []string{"a", "b"},
-		},
-
-		Status: apiv1.ServiceStatus{
-			LoadBalancer: apiv1.LoadBalancerStatus{
-				Ingress: []apiv1.LoadBalancerIngress{
-					apiv1.LoadBalancerIngress{
-						IP: "1.2.3.4",
-					},
-				},
-			},
-		},
-	})
+	svc := testLBService("foo")
+	svc.Spec.ExternalIPs = []string{"a", "b"}
```

Comment:
bit confusing these are a and b and not real IPs.

Reply:
agree 👍  I've updated them to look more like IPs.